core/src/main/java/org/mule/runtime/core/internal/policy/AbstractCompositePolicy.java

```diff
@@ -67,7 +67,7 @@ public AbstractCompositePolicy(List<Policy> policies,
    */
   public final Publisher<Event> processPolicies(Event operationEvent) {
     return just(operationEvent)
-        .flatMapMany(event -> processWithChildContext(event, new AbstractCompositePolicy.NextOperationCall()));
+        .flatMapMany(event -> processWithChildContext(event, new AbstractCompositePolicy.NextOperationCall(), null));
```

Comment:
I added `ComponentLocation` initially just for logging as when a child context is used it's valuable to know who create it (i.e. what component). I was expecting that in some cases it wouldn't be available.  Is there a NullComponentLocation that can be used here instead?